website/client-old/js/services/questServices.js

```diff
@@ -104,6 +104,9 @@ angular.module('habitrpg')
             Groups.data.party = party;
             $state.go('options.social.party');
             resolve();
+            if ($state.current.name === "options.social.party") {
+              $state.reload();
```

Comment:
I bet this is being called from the modal, so it's out of the context from the party controller. Doing the logic here is fine. I'd prefer to just fetch the part object though, rather than do a reload.